doc/install/install_python.rst

```diff
@@ -0,0 +1,103 @@
+.. include:: ../links.inc
+
+.. _install-python:
+
+Installing Python
+=================
+
+MNE-Python runs within Python, and depends on several other Python packages.
+Version |version| requires Python version |min_python_version| or higher. We
+recommend the `Anaconda`_ distribution of Python, which comes with more than
```

Comment:
```suggestion
recommend the `Anaconda`_ Python distribution, which comes with more than
```

Reply:
-1 on this suggestion. Saying "the Anaconda distribution of Python" is, I think, subtly clearer about the fact that what you're getting is Python, from a particular source.  Saying instead "the Anaconda Python distribution" could be read as saying what you're getting is "Anaconda Python", and subtly implies that "Anaconda Python" may be relevantly different from other Python flavors (by analogy to, say, food or clothing brands).

Reply:
OK, that's fine with me. I'd agree 100% if we were talking about using "Anaconda Python" though, but I was specifically suggestion "Anaconda Python distribution", which IMO conveys this subtle differences you're referring to.

Reply:
Reconciliation proposal: put Anaconda in quotation marks?

```
... the "Anaconda" Python distribution
```